### ISSUE TITLE
Do not go back to login screen by pressing the back button

### ISSUE THREAD

**Issue by renatosistemasvc** [react-navigation/react-navigation#5405]
The application returns to the login screen when you click the back button. I need it to simply close the application. The user opens the login screen to login. Once logged in, he can no longer return to the login screen. When you click the back button, it must exit the application. 

I already searched for a solution on the web and did not find it. I found a way to clear the stack, but it only works if the route is inside StackNavigator. In my case the route is inside DrawerNavigator.

**my configuration**

```
const Drawer = DrawerNavigator(
  {
    Login: { screen: Login, navigationOptions:({navigation})=>({ drawerLockMode:"locked-closed" })},
    ListCliente: { screen: ListCliente },  
    Sincronizacao: { screen: Sincronizacao },

  },
  {
    initialRouteName: "Login",
    contentOptions: {
      activeTintColor: "#e91e63"
    },
    contentComponent: props => <SideBar {...props} />
  }
);

const AppNavigator = StackNavigator(
  {
    Drawer: { screen: Drawer }, 
    FormCliente: { screen: FormCliente },
    ListAtendimento: { screen: ListAtendimento },
    FormAtendimento: { screen: FormAtendimento },
  },
  {
    initialRouteName: "Drawer",
    headerMode: "none"
  }
);
```
A possible solution. But it only works if the route is inside StackNavigator. And the CustomerClient route is inside DrawerNavigator. 

```
import { StackActions, NavigationActions } from 'react-navigation';

const resetAction = StackActions.reset({
  index: 0,
  actions: [NavigationActions.navigate({ routeName: 'Profile' })],
});
this.props.navigation.dispatch(resetAction);
```

**summing up. I need to exit the login screen and go to the Customer list screen and not return to the login screen when I click the back button. Anyone know if they have this feature?**

**Comment by brentvatne** [comment]
hi there! this is a question which is better suited for stackoverflow or discord. see https://reactnavigation.org/help.html for more information on how to get help.
